sky/optimizer.py

```diff
@@ -908,6 +909,14 @@ def sort_key(row, accelerator_spot_list=accelerator_spot_list):
             table.add_rows(rows)
             logger.info(f'{table}\n')
 
+            # Warning message for using disk_tier=ultra
+            # TODO(Conless): Consider price of disks in optimizer and move this warning there. # pylint: disable=line-too-long
```

Comment:
Can we separate this into 2 lines to avoid using pylint disable? That should only be used w/ long links or bash commands that cannot be separated into multiple lines.